recipe/meta.yaml

```
@@ -15,7 +15,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1002
+  number: 1004
```

```suggestion
  number: 1003
```
Bumping once per PR is fine/preferred since the intermediate commits won't issue new builds, only the final (merge) commit.

---

Thank's for telling me. I had in mind that this would be necessary each time.